tests/dtm/test_single_process_restart.py

```diff
@@ -155,6 +155,12 @@ def test_read_during_m0d_restart(self):
             proc_read_op.join()
         resp = que.get()
         assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+
+        self.log.info("Step 6: Perform Delete Operations on data written in Step 1:")
```

Comment:
@priyankaborawake  do we want to shift deletes in teardown if that is not the main intetion of the test? And as discussed we can make delete conditional as to delete only objects or even buckets depending on dtm flag. Will be applicable for all tests where delete is not intention of test
cc: @ApurwaMohite  @kulrk

Reply:
Agreed @sonalk0209, 
Separate PR will be raised to move all the delete function for all tests to teardown. 
Except for test where deletes operation are to be main intention of  test.